_layouts/use-case.html

```diff
@@ -52,6 +72,26 @@ <h4 class="text-uppercase d-inline-block ps-3"><a href="{{ page.company_url | de
           </div>
         </div>
 
+        <div class="col-lg-2 d-flex col-6 order-last align-items-center navigation">
+          {% if page.next.url %}
+          <div class="row ps-2 mb-4 m-lg-0">
+            <div class="col">
+              <a href="{{page.next.url}}" class="h-100 m-auto d-flex align-items-center">
+                <small class="text-end">
+                  <span class="text-muted">Next: </span>
```

Comment:
"Next" and "Previous" seem redundant to me as we also show ri-arrow... i would rather use "Use Case:" or just show the project title.

rest looks good. it's nice that the boxes move to the bottom if the screen is too small.

Reply:
Removed. Added titles to help with accessibility.